Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/LocalViewBookmarkLoader.cpp

```diff
@@ -276,19 +269,26 @@ namespace AzToolsFramework
         if (!AZ::SettingsRegistryMergeUtils::DumpSettingsRegistryToStream(*registry, "", stringStream, CreateDumperSettings(bookmarkKey)))
         {
             AZ_Error(
-                "LocalViewBookmarkLoader", false, R"(Unable to dump SettingsRegistry to stream "%.*s"\n)", AZ_STRING_ARG(localBookmarksFileName.Native()));
+                "LocalViewBookmarkLoader",
+                false,
+                R"(Unable to dump SettingsRegistry to stream "%.*s"\n)",
+                AZ_STRING_ARG(localBookmarksFileName.Native()));
             return;
         }
 
         [[maybe_unused]] const bool saved = m_streamWriteFn(
-            localBookmarksFileName, stringBuffer,
+            localBookmarksFileName,
+            stringBuffer,
             [this](AZ::IO::GenericStream& genericStream, AZStd::string_view stringBuffer)
             {
                 return Write(genericStream, stringBuffer);
             });
 
         AZ_Warning(
-            "LocalViewBookmarkLoader", saved, R"(Unable to save Local View Bookmark file to path "%.*s"\n)", AZ_STRING_ARG(localBookmarksFileName.Native()));
+            "LocalViewBookmarkLoader",
+            saved,
+            R"(Unable to save Local View Bookmark file to path "%.*s"\n)",
+            AZ_STRING_ARG(localBookmarksFileName.Native()));
```

Comment:
AZ_Warning is defined as nothing when AZ_ENABLE_TRACING is not set.  Its unset in RELEASE.  Which is maybe fine for Tools Framework code, if we ever want to ship it in release, though, this would be an issue.

This will cause this not to actually try to save if its ever hit in release.   Is this ok?  I assume this is why you added [[maybe_unused]] so it might be okay depending on what/when you expect this code to run.  It looks like its actually doing work though.

Reply:
I'll look into this and double check. I think this is likely a problem we'd hit in lots of code in AzToolsFramework as I don't think we ever compile it in Release and so don't hit these unused variable warnings.